### ISSUE TITLE
Confusing error message when selecting a DBL portion

### ISSUE THREAD

**Issue by sewhite** [sillsdev/ptx2pdf#831]
Trying out this great new feature of downloading a bundle from the DBL.

I picked the Akukem project from the DBL list #1. 
I selected the zip and told PTXPrint to unpack it.
It unpacked, then said the copyright statement in Paratext was too long, and I needed to shorten it. I The error message did not tell me how to fix this when I did not have it as a Paratext project. 
When I closed the error message and tried to print anyway, it did make a  PDF.

I still cannot edit the copyright statement, because although I have an Akukem folder in My Paratext 9 Projects, Paratext 9 does not show it in the list of projects. 
[Akukem-DefaultPTXprintArchive.zip](https://github.com/sillsdev/ptx2pdf/files/10747956/Akukem-DefaultPTXprintArchive.zip)

**Comment by sewhite** [comment]
I realized I did not have the latest versions, so I downloaded 2.2.50 and tried again. Same result
![image](https://user-images.githubusercontent.com/16904461/219172464-7aa10e00-7139-4356-bcd4-b672ba9fd4f3.png)

**Comment by mnjames** [comment]
Issue #799 gives some background to this. PTXprint can't handle complex html in that field. Giving a warning and then ignoring that field seems like a reasonable workaround.

If you really want to get rid of the error, after downloading the DBL bundle go into the .zip file, open the `metadata.xml` file, find the copyright statement, and remove the multiple `<p>...</p>` lines so that there is just one line. After that it should open in PTXprint without complaining.

**Comment by sewhite** [comment]
Thanks for your response. Since PTXPrint had already unpacked the archive, I found I could go into the ptxsettings.xml file, remove the || strings in the copyright statement, and then I did not get the error message.  Steve From: mnjamesSent: Thursday, February 16, 2023 3:17 AMTo: sillsdev/ptx2pdfCc: Steve White; AuthorSubject: Re: [sillsdev/ptx2pdf] Confusing error message when selecting a DBL portion (Issue #831) Issue #799 gives some background to this. PTXprint can't handle complex html in that field. Giving a warning and then ignoring that field seems like a reasonable workaround.If you really want to get rid of the error, after downloading the DBL bundle go into the .zip file, open the metadata.xml file, find the copyright statement, and remove the multiple <p>...</p> lines so that there is just one line. After that it should open in PTXprint without complaining.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
